l3kernel/l3sys.dtx

```diff
@@ -514,6 +524,18 @@
 % \end{variable}
 % \end{macro}
 %
+% \begin{function}[pTF]{\sys_if_engine_opentype:}
+%    \begin{macrocode}
+\@@_const:nn
+  { sys_if_engine_opentype }
+  {
+    \bool_lazy_or_p:nn
```

Comment:
This is defining `_engine_opentype` as being luatex or xetex, which isn't what I expected. I thought this would be a feature test testing for a suitable `\U` primitive such as `\Uchardef` not a named engine test

Reply:
Well yes, but that's back with it's hard to decouple the ideas anyway

Reply:
no test is perfect but I think making the test a feature test like choosing the name makes it clear that it is a test for the font support being there and isn't just a named engine list, and practically you want this to default as true in luatex or xetex variants (which may or may not be close enough to have the luatex or xetex tests being `true`) But mainly I think it is for documentation and messaging rather than practical difference